src/components/HowSoonOptions.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import '../css/components/HowSoonOptions.css';
+
+const HowSoonOptions = () => (
+  <fieldset className="fieldset">
+    <legend className="legend">How soon will you buy this item?</legend>
+    {howSoonData.map((obj) => (
+      <RadioButton type="radio" name="how-soon" key={obj.id} {...obj} />
+    ))}
+  </fieldset>
```

Comment:
Amazing work on using `<fieldset>` & `<legend>` tags here for accessibility purposes. This is something new I just learned from you all. 👍👏